lib/pf/config/util.pm

```diff
@@ -627,7 +628,15 @@ sub send_using_smtp_callback {
         and !$args{NoAuth} )
     {
         if ( $smtp->supports( 'AUTH', 500, ["Command unknown: 'AUTH'"] ) ) {
-            $smtp->auth( $args{AuthUser}, $args{AuthPass} )
+            my $sasl = Authen::SASL->new(
+              mechanism => 'LOGIN PLAIN',
```

Comment:
@jrouzierinverse Any updates on my comments?

Reply:
By doing some source diving it seems to be the case.
It will attempt them in order removing which one failed and retry.
You can see how it used here.
https://metacpan.org/pod/Authen::SASL

Reply:
Ok good, thanks for the explanation. Just flip LOGIN and PLAIN then, since PLAIN is a lot more popular.

Reply:
What about the timeout? In the bug report, it's mentioned that is is not used and should be.